modules/i18n/src/translations/en-US/meta.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Update the license here as follows, as you are updating the file.
```suggestion
 * Copyright (c) 2020-2024, WSO2 LLC. (https://www.wso2.com).
```

Reply:
Follow this when moving forward.